Documentation/kubernetes-on-baremetal.md

```diff
@@ -0,0 +1,33 @@
+# Kubernetes Installation on Bare Metal &amp; CoreOS
+
+This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS.
+After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using he kubectl CLI tool.
+
+## Deployment Prerequisites
+
+### CoreOS Installation
+
+For all nodes running Kubernetes components (master & workers), you must use CoreOS version 773.1.0+ for the kubelet to be present in the image.
+
+Use the official CoreOS Bare Metal guides for installation instructions:
+
+* [Booting with iPXE][coreos-ipxe]
+* [Booting with PXE][coreos-pxe]
+* [Installing to Disk][coreos-ondisk]
+
```

Comment:
"Mixing multiple methods is possible. For example, doing an install to disk for the machines running the etcd cluster and Kubernetes Masters, but PXE-booting the worker machines."

This might encourage folks to dig into each guide, and go into into more of a creative, brainstorming mode instead of "quickly pick a guide".